### ISSUE TITLE
[BUG] Wrong letter highlighted in `disks` panel

### ISSUE THREAD

**Issue by lepa22** [aristocratos/bpytop#246]
**Describe the bug**

In the `disks` panel of the interface, the letter "o" of `io` is highlighted (screenshot 1), signifying that it is a toggle for a setting of this panel. However, pressing <kbd>o</kbd> opens a settings menu (screenshot 2), while the actual toggle for this panel is "i" (screenshot 3).

**Screenshots**

1.

![Screenshot_2021-01-16_00-29-13](https://user-images.githubusercontent.com/16370046/104784994-230f9e80-5792-11eb-93b5-b2809af6d323.png)

2.

![Screenshot_2021-01-16_00-37-53](https://user-images.githubusercontent.com/16370046/104785398-16d81100-5793-11eb-8730-d48d7c31fa2a.png)

3.

![Screenshot_2021-01-16_00-39-42](https://user-images.githubusercontent.com/16370046/104785521-60286080-5793-11eb-904a-fd4b2f973575.png)

**Info:**
 - bpytop version: `1.0.59`
 - psutil version: `5.6.2`
 - Linux distribution and version: Ubuntu 20.04
 - Terminal used: kitty
 - Font used: Fira Code
 - Python version: 3.7.9

**Additional context**

contents of `~/.config/bpytop/error.log`:

```
16/01/21 (00:18:45) | WARNING: psutil version 5.6.2 detected, version 5.7.0 or later required for full functionality!
16/01/21 (00:18:45) | ERROR: Non fatal error during disk io collection!
16/01/21 (00:18:45) | ERROR: Caused by outdated psutil version.
16/01/21 (00:18:45) | ERROR: not sure how to interpret line '   7       0 loop0 306 0 3248 95 0 0 0 0 0 120 95 0 0 0 0 0 0\n'
Traceback (most recent call last):
  File "/usr/local/bin/bpytop", line 3207, in _collect
    io_counters = psutil.disk_io_counters(perdisk=SYSTEM != "BSD", nowrap=True)
  File "/home/lepa/.local/lib/python3.7/site-packages/psutil/__init__.py", line 2169, in disk_io_counters
    rawdict = _psplatform.disk_io_counters(**kwargs)
  File "/home/lepa/.local/lib/python3.7/site-packages/psutil/_pslinux.py", line 1114, in disk_io_counters
    for entry in gen:
  File "/home/lepa/.local/lib/python3.7/site-packages/psutil/_pslinux.py", line 1087, in read_procfs
    raise ValueError("not sure how to interpret line %r" % line)
ValueError: not sure how to interpret line '   7       0 loop0 306 0 3248 95 0 0 0 0 0 120 95 0 0 0 0 0 0\n'
```

**Comment by aristocratos** [comment]
@lepa22 
Yup, forgot to switch back the highlighting to `i`, will fix in next update.

**Comment by lepa22** [comment]
Thank you for your quick response and your great work!

**Comment by aristocratos** [comment]
@lepa22 
Fixed in v1.0.60
